ozy/export.py

```diff
@@ -1,8 +1,8 @@
 import numpy as np
 from ozy.dict_variables import common_variables,grid_variables,particle_variables,get_code_units,basic_conv
 import sys
-sys.path.append('/mnt/zfsusers/currodri/Codes/ozymandias/ozy/amr')
-sys.path.append('/mnt/zfsusers/currodri/Codes/ozymandias/ozy/part')
+sys.path.append('/home/currodri/Codes/ozymandias/ozy/amr')
+sys.path.append('/home/currodri/Codes/ozymandias/ozy/part')
```

Comment:
This is not necessary and should be done as the rest of modules: loaded in a general way through the PATH